src/stats/violin.jl

```diff
@@ -47,10 +48,17 @@ end
 function plot!(plot::Violin)
     x, y = plot[1], plot[2]
     args = @extract plot (width, side, color, show_median, npoints, boundary, bandwidth,
-        datalimits, max_density, dodge, n_dodge, gap, dodge_gap)
-    signals = lift(x, y, args...) do x, y, width, vside, color, show_median, n, bound, bw, limits, max_density, dodge, n_dodge, gap, dodge_gap
+        datalimits, max_density, dodge, n_dodge, gap, dodge_gap, orientation)
+    signals = lift(x, y, args...) do x, y, width, vside, color, show_median, n, bound, bw, limits, max_density, dodge, n_dodge, gap, dodge_gap, orientation
         x̂, violinwidth = compute_x_and_width(x, width, gap, dodge, n_dodge, dodge_gap)
 
+        # for horizontal violin just flip all componentes
+        fpoint, frect = Point2f, Rectf
```

Comment:
Can we revert that change? Seems unecessary and seems to degrade readability ;)

Reply:
Looks like it's taken from [the crossbar recipe](https://github.com/JuliaPlots/Makie.jl/blob/7c148cd785ca5e5e9ade82c755d48263d7741125/src/stats/crossbar.jl#L67), but I agree that it decreases readability (both here and there). @SirLukeSchande, maybe something like

```julia
if orientation == :horizontal
    verts .= _flip_xy.(verts)
end
```

(and analogous for the median endpoints) would be easier to read. See eg [here](https://github.com/JuliaPlots/Makie.jl/blob/7c148cd785ca5e5e9ade82c755d48263d7741125/src/stats/boxplot.jl#L154).

Reply:
It was taken from crossbar. You are right, that the barplot implementation is more readable. Do I open a new Pull request for this?